README.md

```diff
@@ -40,6 +40,34 @@ Option     | Default value     | Description
 `api_key`  |                   | Mandrill API key.
 `async`    | `false`           | Enable a background sending mode that is optimized for bulk sending.
 
+### Mandrill Templates
+
+If you want to use this gem with mandrill templates you just have to add the `template` param to the `mail` function.
+
+```ruby
+class MyMailer < ActionMailer::Base
+  def notify_user(email)
+    mail(
+      to: email,
+      from: 'your@email.com',
+      template: 'your-mandrill-template-slug',
+      template_content: [ # optional
+        {
+          name: 'header',
+          content: 'string to replace a *|header|* in your template', 
+        },
+        {
+          name: 'content',
+          content: 'string to replace a *|content|* in your template'
+          # OR
+          content: render_to_string # to use the contents in app/views/my_mailer/notify_user.html.erb
```

Comment:
It may confuse some newbies and they will come here with stupid questions. I don't want that and there's no point in listing all possible ways of building a string - obviously everyone will use some methods, not just plain strings.